src/bot/api.rs

```diff
@@ -17,10 +19,10 @@ impl Bot {
         GetMe::new(self.ctx())
     }
 
-    pub fn send_message<C, T>(&self, chat_id: C, text: T) -> SendMessage
+    pub fn send_message<'a, C, T>(&'a self, chat_id: C, text: T) -> SendMessage
     where
-        C: Into<ChatId>,
-        T: Into<String>,
+        T: Into<Cow<'a, str>>,
+        C: Into<ChatId<'a>>,
     {
         SendMessage::new(self.ctx(), chat_id.into(), text.into())
```

Comment:
`SomeRequest::new` accept now, so there is no need to call `.into()`
```suggestion
        SendMessage::new(self.ctx(), chat_id, text)
```